src/BenchmarkDotNet/Code/CodeGenerator.cs

```diff
@@ -95,6 +97,24 @@ internal static string Generate(BuildPartition buildPartition)
             return benchmarkProgramContent;
         }
 
+        private static string GetCustomEnvInfo(DeclarationsProvider provider)
+        {
+            if (provider.CustomEnvInfoMethodName is null)
+            {
+                return "Array.Empty<string>()";
+            }
+
+            switch (provider.CustomEnvInfoMethodReturnType)
+            {
+                case Type t when t == typeof(string):
+                    return $"new string[] {{ {provider.CustomEnvInfoMethodName}() }}";
+                case Type t when typeof(IEnumerable<string>).IsAssignableFrom(t):
+                    return $"{provider.CustomEnvInfoMethodName}()";
+                default:
+                    return "Array.Empty<string>()";
```

Comment:
Could you please use the full name?

```suggestion
                    return "System.Array.Empty<string>()";
```